pymc3/tests/test_distributions_random.py

```diff
@@ -188,10 +188,9 @@ def get_random_variable(self, shape, with_vector_params=False, name=None):
         @staticmethod
         def sample_random_variable(random_variable, size):
             """ Draws samples from a RandomVariable using its .random() method. """
-            if size is None:
-                return random_variable.eval()
-            else:
-                return change_rv_size(random_variable, size, expand=True).eval()
+            if size:
```

Comment:
Can we make this explicit?

Reply:
With the new robustness in `change_rv_size` this is just fine.